src/main/java/no/nav/familie/integrasjoner/client/rest/OppgaveRestClient.kt

```diff
@@ -33,12 +33,23 @@ class OppgaveRestClient(@Value("\${OPPGAVE_URL}") private val oppgaveBaseUrl: UR
 
     private val LOG = LoggerFactory.getLogger(OppgaveRestClient::class.java)
 
-    fun finnOppgave(request: Oppgave): OppgaveJsonDto {
-        val requestUrl = lagRequestUrlMed(request.aktorId, request.journalpostId, request.tema?.name ?: KONTANTSTØTTE_TEMA)
-        return requestOppgaveJson(requestUrl)
+    fun finnOppgave(request: Oppgave): Oppgave {
+        when {
+            request.aktoerId == null -> {
```

Comment:
Kan det være det endelig er en usecase for takeUnless?  😃

    request.takeUnless { it.aktorId == null } ?: error(...